controllers/etcdpeer_controller.go

```diff
@@ -16,14 +20,104 @@ type EtcdPeerReconciler struct {
 	Log logr.Logger
 }
 
+func ignoreNotFound(err error) error {
+	if apierrs.IsNotFound(err) {
+		return nil
+	}
+	return err
+}
```

Comment:
There is also a `client.IgnoreNotFound` function if you'd prefer to not redefine this 😄

Reply:
Yes, yes I would. Blame the Kubebuilder examples again (are they out of date?)